config/saml.yml

```diff
@@ -16,3 +17,4 @@ production:
   metadata_url: <%= ENV["METADATA_URL"] %>
   certificate: <%= File.read(File.expand_path(ENV["CERTIFICATE_FILE"])).dump %>
   key: <%= File.read(File.expand_path(ENV["KEY_FILE"])).dump %>
+  relay_url: <%=ENV["RELAY_URL"] %>
```

Comment:
I'm actually not familiar with where these ENV vars are created.  I'm assuming somewhere in travis.  This will need to be created.  It represents the base url for the vets-website.